### PR TITLE
Install Script

### PR DESCRIPTION
**Pull request by chigozienri**
Proposal for install.sh to reduce friction when installing cog

Run as follows:
```console
# via curl
sh -c "$(curl -fsSL https://raw.githubusercontent.com/replicate/cog/main/tools/install.sh)"
# or via wget:
sh -c "$(wget -qO- https://raw.githubusercontent.com/replicate/cog/main/tools/install.sh)"
# or via fetch:
sh -c "$(fetch -o - https://raw.githubusercontent.com/replicate/cog/main/tools/install.sh)"
```

See Slack comments here https://replicatehq.slack.com/archives/C05600FDYTE/p1709142996116999

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,7 +121,17 @@ If you're using macOS, you can install Cog using Homebrew:
 brew install cog
 ```
 
-You can also download and install the latest release of Cog
+You can run an install script which will download and install the latest release of Cog:
+```
+# via curl
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/replicate/cog/main/tools/install.sh)"
```
Comment (zeke) [inline README.md:127]: Would `curl {URL} | sh` also work?

That might be a bit cleaner, but maybe there are some tradeoffs there?

Comment (zeke) [reply README.md:127]: Could also use a short URL like `https://replicate.fyi/install-cog` with a redirect to this longer URL.

Comment (zeke) [reply README.md:127]:
```
curl -fsSL https://cog.run/installer | sh
```

File: README.md [hunk 2]
```diff
@@ -121,7 +121,17 @@ If you're using macOS, you can install Cog using Homebrew:
...
+# or via wget:
+sh -c "$(wget -qO- https://raw.githubusercontent.com/replicate/cog/main/tools/install.sh)"
+# or via fetch:
+sh -c "$(fetch -o - https://raw.githubusercontent.com/replicate/cog/main/tools/install.sh)"
```
Comment (chigozienri) [inline README.md:131]:
```suggestion
curl -fsSL https://cog.run/installer | sh
# or via wget:
wget -qO- https://cog.run/installer | sh
# or via fetch:
fetch -o - https://cog.run/installer | sh
```

@zeke I don't know where to make a PR to update cog.run, do you know? This does look nicer, though.

File: README.md (hunk 1 above)
Comment (chigozienri) [reply README.md:127]: Yes, for some reason I thought the construction using a pipe wouldn't work with wget, but I just tested and it does.

(N.B. Part of the motivation for making this script in the first place was that my friend tried to use the existing instructions and one of his frustrations was that Ubuntu didn't have curl installed by default.)

File: README.md (hunk 2 above)
Comment (zeke) [reply README.md]: @chigozienri I think we'd need to set up a custom route handler for that in CloudFlare

Comment (zeke) [reply README.md]: I set that up:

```
$ curl -I "https://cog.run/installer"
HTTP/2 301 
date: Tue, 19 Mar 2024 02:49:41 GMT
location: https://raw.githubusercontent.com/replicate/cog/main/tools/install.sh
```

File: README.md [hunk 3]
```diff
@@ -121,7 +121,17 @@ If you're using macOS, you can install Cog using Homebrew:
 brew install cog
 ```
 
-You can also download and install the latest release of Cog
+You can run an install script which will download and install the latest release of Cog:
```
Comment (mattt) [inline README.md:124]:
```suggestion
You can also download and install the latest release using our [install script](https://cog.run/installer):

```

File: README.md [hunk 4]
```diff
@@ -121,7 +121,18 @@ If you're using macOS, you can install Cog using Homebrew:
 brew install cog
 ```
 
-You can also download and install the latest release of Cog
+You can also download and install the latest release using our [install script](https://cog.run/installer):
```
Comment (mattt) [inline README.md:124]:
```suggestion
You can also download and install the latest release using our [install script](https://cog.run/install):
```

File: README.md [hunk 5]
```diff
@@ -121,7 +121,18 @@ If you're using macOS, you can install Cog using Homebrew:
...
+
+```
+# via curl
+curl -fsSL https://cog.run/installer | sh
+# or via wget:
+wget -qO- https://cog.run/installer | sh
+# or via fetch:
+fetch -o - https://cog.run/installer | sh
```
Comment (mattt) [inline README.md:132]:
```suggestion
# via curl
curl -fsSL https://cog.run/install | sh
# or via wget:
wget -qO- https://cog.run/install | sh
# or via fetch:
fetch -o - https://cog.run/install | sh
```

File: README.md [hunk 6]
```diff
@@ -121,7 +121,18 @@ If you're using macOS, you can install Cog using Homebrew:
 brew install cog
 ```
 
-You can also download and install the latest release of Cog
+You can also download and install the latest release using our [install script](https://cog.run/install):
```
Comment (mattt) [inline README.md:124]: Also resolves from https://cog.run/install.sh

**Review by mattt — APPROVED** [review]

**Comment by zeke** [comment]
Yayy!

**Pull request merged**